mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -1514,7 +1515,9 @@ class MqttSessionSpec
       server.watchCompletion().foreach(_ => session.shutdown())
     }
 
-    "close when no ping request received" ignore { // assertAllStagesStopped { // https://github.com/akka/alpakka/issues/1563
+    "close when no ping request received" in assertAllStagesStopped {
+      implicit val patienceConfig = PatienceConfig(scaled(Span(3000, Millis)), scaled(Span(15, Millis)))
```

Comment:
Can you please explain why we need this patience config? Is there a way to avoid it? I’m not overly concerned, but we’ve not had to use such config so far.

Reply:
The reason is that the test is using 1 second keep alive parameter so using default PatienceConfig of 150 milliseconds as max value is not enough for the ping timeout to trigger. Therefore I'm adding 3 seconds max time before I see the stream failing because of the ping timeout or otherwise failing.

Reply:
How about reducing the ping timeout instead?